tasks/templates/tasks/view_single_task.html

```diff
@@ -18,12 +18,12 @@ <h2>Tasks</h2>
                     </div>
                     <h3><a class="article_title" href="#">{{ task.title }}</a></h3>
                     <p class="article-content">{{ task.description }}</p>
-                    {% if  user.is_manager %}
-					 	          <div>
-					 	          	<a class="btn btn-secondary btn-sm mt-1 md-1" href="#">Update</a>
-						            <a class="btn btn-danger btn-sm mt-1 md-1" href="#">Detele</a>
-					 	          </div>
-				          	{% endif %}
+                     {% if  user.is_manager %}
```

Comment:
I meant the redundant space between the "if" to "user.is_manager"
```suggestion
                     {% if user.is_manager %}
```